pyproject.toml

```diff
@@ -154,20 +154,149 @@ warn_return_any = true
 follow_imports = "silent"
 exclude = [
     # As we fix more and more of these, we should remove them from the list
-    "llama_stack/providers",
-    "llama_stack/distribution",
-    "llama_stack/apis",
-    "llama_stack/cli",
-    "llama_stack/models",
-    "llama_stack/strong_typing",
-    "llama_stack/templates",
+    "^llama_stack/apis/agents/agents\\.py$",
+    "^llama_stack/apis/batch_inference/batch_inference\\.py$",
+    "^llama_stack/apis/benchmarks/benchmarks\\.py$",
+    "^llama_stack/apis/common/content_types\\.py$",
+    "^llama_stack/apis/common/training_types\\.py$",
+    "^llama_stack/apis/datasetio/datasetio\\.py$",
+    "^llama_stack/apis/datasets/datasets\\.py$",
+    "^llama_stack/apis/eval/eval\\.py$",
+    "^llama_stack/apis/files/files\\.py$",
+    "^llama_stack/apis/inference/inference\\.py$",
+    "^llama_stack/apis/inspect/inspect\\.py$",
+    "^llama_stack/apis/models/models\\.py$",
+    "^llama_stack/apis/post_training/post_training\\.py$",
+    "^llama_stack/apis/resource\\.py$",
+    "^llama_stack/apis/safety/safety\\.py$",
+    "^llama_stack/apis/scoring/scoring\\.py$",
+    "^llama_stack/apis/scoring_functions/scoring_functions\\.py$",
+    "^llama_stack/apis/shields/shields\\.py$",
+    "^llama_stack/apis/synthetic_data_generation/synthetic_data_generation\\.py$",
+    "^llama_stack/apis/telemetry/telemetry\\.py$",
+    "^llama_stack/apis/tools/rag_tool\\.py$",
+    "^llama_stack/apis/tools/tools\\.py$",
+    "^llama_stack/apis/vector_dbs/vector_dbs\\.py$",
+    "^llama_stack/apis/vector_io/vector_io\\.py$",
+    "^llama_stack/cli/download\\.py$",
+    "^llama_stack/cli/llama\\.py$",
+    "^llama_stack/cli/stack/_build\\.py$",
+    "^llama_stack/cli/stack/list_providers\\.py$",
+    "^llama_stack/distribution/build\\.py$",
+    "^llama_stack/distribution/client\\.py$",
+    "^llama_stack/distribution/configure\\.py$",
+    "^llama_stack/distribution/library_client\\.py$",
+    "^llama_stack/distribution/request_headers\\.py$",
+    "^llama_stack/distribution/routers/",
```

Comment:
Note: I have to leave some of these as directories because, apparently, `exclude` doesn't know how to handle `__init__.py` files that violate rules.

Reply:
is it worth a comment in the file?